### ISSUE TITLE
New contributor speedbumps

### ISSUE THREAD

**Issue by thatch** [facebookincubator/Bowler#36]
* isort wants to change imports on a fresh checkout (should we do another run of black after?)
* mypy requires a python3.6 executable, Arch only has 3.7 by default

**Comment by jreese** [comment]
We should just drop the --python-version argument from the mypy check, and just assume that users are running 3.6+ (because otherwise bowler's fstrings will stop them).  That argument is just a copy pasta from other projects.

**Comment by thatch** [comment]
I agree dropping python-version would be great.  (I'm no longer blocked, but we can keep this open until it's gone.)
